.circleci/config.yml

```diff
@@ -24,19 +24,14 @@ jobs:
           name: Mutect2 vs Strelka2
           command: ./scripts/run_in_ci.sh Rscript -e "rmarkdown::render('analyses/mutect2-vs-strelka2/01-set-up.Rmd', clean = TRUE);
                                                       rmarkdown::render('analyses/mutect2-vs-strelka2/02-analyze-concordance.Rmd', clean = TRUE)"
-      
+     
       - run:
           name: fusion filtering
-          command: ./scripts/run_in_ci.sh Rscript analyses/fusion_filtering/01-fusion-standardization.R -f "data/pbta-fusion-starfusion.tsv.gz" -c "STARfusion" -o "scratch/star.RDS" 
-                                                
+          command: ./scripts/run_in_ci.sh Rscript analyses/fusion_filtering/01-fusion-standardization.R -f "data/pbta-fusion-arriba.tsv.gz" -c "arriba" -o "scratch/arriba.tsv" 
+ 
       - run:
-          name: Unsupervised Transcriptomic Analysis - Subset by Strategy          
-          command: ./scripts/run_in_ci.sh Rscript --vanilla analyses/transcriptomic-dimension-reduction/01-subset-expression-by-strategy.R
-    
-      - run:
-          name: Unsupervised Transcriptomic Analysis - Run Dimension Reduction         
-          command: ./scripts/run_in_ci.sh Rscript --vanilla analyses/transcriptomic-dimension-reduction/scripts/run-dimension-reduction.R  --expression data/pbta-gene-expression-rsem.fpkm.rds --metadata data/pbta-histologies.tsv --filename_lead rsem_all --output_directory analyses/transcriptomic-dimension-reduction/results --skip_tsne
-                
+          name: fusion filtering
+          command: ./scripts/run_in_ci.sh Rscript analyses/fusion_filtering/02-fusion-filtering.R -S scratch/arriba.tsv -e data/pbta-gene-expression-rsem.fpkm.rds -r FALSE -a "NEIGHBORS|GTEx|HGNC_GENEFAM|DGD_PARALOGS|Normal|BodyMap|ConjoinG" -j 1 -s 10 -i "inframe|frameshift|other" -R analyses/fusion_filtering/references/ -o scratch/arriba -t 1                         
```

Comment:
I guess we would have to divide the standardized fusion calls before 02-fusion-filtering.R now that we have stranded/unstanded. I can look into that. What do you and @jharenza think about that?

Reply:
@kgaonkar6 I think it will be easier to merge the RDS files for the expression, rather than subset the files for the the fusions...

Reply:
Strictly from a code point of view - you may want the script to be robust to samples missing from the expression matrix, where you warn users about missing samples and _only_ return/write to file data for samples that are in the expression matrix. That way you could presumably run this twice: 1) fusions + poly-A expression data 2) fusions + stranded data. This also lets you set different thresholds for the expression filter.

I don't know enough about the upstream steps to comment on if the fusion files should also be separated based on strategy.

Reply:
To make it more flexible/applicable ( since this is a general filtering step) I liked @jaclyn-taroni suggestion and so I'm now going to run the polya and stranded rds expressionMatrix separately and then merge in project specific filtering.